src/include/network/services/host.rb

```diff
@@ -81,33 +81,29 @@ def HostsMainDialog(standalone)
           "<p>Enter a host <b>IP Address</b>, a <b>Hostname</b>, and optional\n<b>Host Aliases</b>, separated by spaces.</p>\n"
         )
 
-      max = 0
       table_items = []
       deleted_items = []
       hosts = Host.name_map
 
       Builtins.y2debug("hosts=%1", hosts)
 
       # make ui items from the hosts list
-      Builtins.maplist(hosts) do |host, names|
-        if Ops.less_than(Builtins.size(names), 1)
+      table_items = hosts.map do |host, names|
+        if names.empty?
           Builtins.y2error("Invalid host: %1, (%2)", host, names)
```

Comment:
Please, contribute to kill those deprecated `Builtins` using `Yast::Logger` instead :)

Reply:
;-) there is so many ```Builtins``` that I never know when to stop ;-)